### PR TITLE
Update CLI readme

### PR DESCRIPTION
**Pull request by JackuB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,289 +2,256 @@
...
-    -v "/home/user/.ivy2:/home/node/.ivy2"
-  snyk/snyk-cli:sbt-1.0.4 test --org=my-org-name
-```
+</details>
 
-### Gradle 2.8 / Gradle 4.4 / Gradle 5.4
+## Install as a part of a Snyk CLI integration
 
-[See all snyk/snyk-cli gradle images](https://hub.docker.com/r/snyk/snyk-cli/tags?page=1&name=gradle)
+Snyk also offers many integrations into developer tooling. These integrations will install and manage Snyk CLI for you. For example:
```
Comment (maxjeffos) [inline README.md:141]:
```suggestion
Snyk also offers many integrations into developer tooling. These integrations will install and manage the Snyk CLI for you. For example:
```

File: README.md [hunk 2]
```diff
@@ -2,289 +2,256 @@
...
 ```
-docker run -it
-    -e "SNYK_TOKEN=<TOKEN>"
-    -e "USER_ID=1234"
-    -e "MONITOR=true"
-    -v "<PROJECT_DIRECTORY>:/project"
-    -v "/home/user/.gradle:/home/node/.gradle"
-  snyk/snyk-cli:gradle-4.4 test --org=my-org-name
+
+## Authorization
```
Comment (maxjeffos) [inline README.md:169]: Should we mention additional configuration options (ex using `SNYK_TOKEN`) as well as talk about the `snyk.json` config file?

File: README.md [hunk 3]
```diff
@@ -2,289 +2,256 @@
...
-    -e "MONITOR=true"
-    -v "<PROJECT_DIRECTORY>:/project"
-    -v "/home/user/.gradle:/home/node/.gradle"
-  snyk/snyk-cli:gradle-2.8 test --org=my-org-name
-```
+---
 
+# Getting started with Snyk CLI
+
+Once you installed Snyk CLI, you can verify it's working by running
```
Comment (maxjeffos) [inline README.md:163]:
```suggestion
Once you installed the Snyk CLI, you can verify it's working by running
```

File: README.md [hunk 4]
```diff
@@ -2,289 +2,256 @@
...
-```
-docker run -it
-    -e "SNYK_TOKEN=<TOKEN>"
-    -e "USER_ID=1234"
-    -e "MONITOR=true"
-    -e "TARGET_FILE=requirement-dev.txt"
-    -v "<PROJECT_DIRECTORY>:/project"
-  snyk/snyk-cli:python-3 test --org=my-org-name
-```
+We recommend reaching to [support@snyk.io](mailto:support@snyk.io) email whenever you need help with Snyk CLI or Snyk in general.
```
Comment (maxjeffos) [inline README.md:241]:
```suggestion
We recommend reaching out via the [support@snyk.io](mailto:support@snyk.io) email whenever you need help with Snyk CLI or Snyk in general.
```

File: README.md [hunk 5]
```diff
@@ -2,289 +2,256 @@
...
 
-## Badge
+GitHub Issues on any [Snyk project](https://github.com/snyk) are not actively monitored by Snyk support.
 
-Make users feel more confident in using your website by adding your Snyk badge!
+## Contributing
 
-<a href="https://snyk.io/test/github/snyk/snyk"><img src="https://snyk.io/test/github/snyk/snyk/badge.svg" alt="Known Vulnerabilities"/></a>
+This project is open source, but we don't encourage outside contributors.
```
Comment (maxjeffos) [inline README.md:247]:
```suggestion
This project is open source but we don't encourage outside contributors.
```

**Review by Avishagp — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (JackuB) [reply README.md:169]: Added to the CI/CD section

**Pull request merged**